src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/SchemaExporter.cs

```diff
@@ -352,7 +352,8 @@ private XmlElement ExportGenericInfo(Type clrType, string elementName, string el
                 ignoreExtensionDataObject: false, preserveObjectReferences: true);
             serializer.WriteObject(xmlWriter, surrogateData);
             xmlWriter.Flush();
-            return (XmlElement?)XmlDoc.ReadNode(XmlReader.Create(new StringReader(stringWriter.ToString())));
+            using var xmlReader = XmlReader.Create(new StringReader(stringWriter.ToString()));
```

Comment:
I think this usage of `XmlReader.Create(Stream input)` will not dispose the underlying stream.

`StringReader` doesn't do a whole lot on dispose aside from reset position and null the underlying array...

Might still be worth it however?

Reply:
Right, there's no functional benefit to this change; there are no resources here to Dispose.